tests/common/lib/env.bash

```diff
@@ -22,9 +22,14 @@ env.init() {
 
   ck8s init both
 
+  yq_set 'common' '.global.ck8sCloudProvider' "\"${CK8S_CLOUD_PROVIDER}\""
+
```

Comment:
Isn't this set in the default config?

Reply:
Not sure, I was having trouble getting the tests to pass, this was one of the fields claimed to be wrong in the validation step.